modules/ROOT/pages/restarting-a-node.adoc

```diff
@@ -1,78 +1,118 @@
-= Stop, Restart, or Update an Anypoint Private Cloud Edition Node
+= Update Nodes
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic describes the processes for removing an Anypoint Platform PCE node from active service, and returning it to service when ready.
+== Before You Begin
 
-== Notes
+* Set up and test the backup procedure on all clusters.
+* Perform a backup before attempting node maintenance to preserve data in the event of a failure.
 
-[NOTE]
-To preserve your data in case of system failure, ensure that your system is backed up before attempting node maintenance.
-[NOTE]
-The steps below must be performed on a single Anypoint Platform PCE node at a time.  If maintenance is required on more than one node, the entire process must be completed on a single node before the next node is addressed.
+== Cluster and Application Updates
 
-== Drain the Node
+[IMPORTANT]
+The following steps must be performed on one Anypoint Platform Private Cloud Edition node at a time to minimize or eliminate component breakage during the update process. The entire process must be completed on each node before the next node is addressed. Verify the health status of all cluster components and applications after each node update before proceeding with the next node.
```

Comment:
```suggestion
You must perform cluster and application updates on one Anypoint Platform Private Cloud Edition node at a time to minimize or eliminate component breakage during the update process. You must complete the entire process and verify the health of all cluster components on one node before proceeding to the next node.
```

Reply:
BUT...this intro text names user actions in a different order than the following three tasks are presented.
Please make the order in the intro and the three subtopics match.

Reply:
In fact, consider simplifying the intro text as follows:
```suggestion
To make cluster and application updates, you must perform the following tasks completely and in order, one node at a time:
. Verify the status of cluster health.
. Install updates.
. Verify installation success.
```